care/facility/api/serializers/prescription.py

```diff
@@ -98,6 +99,13 @@ class Meta:
         )
 
     def validate(self, attrs):
+        def extract_numeric_value(dosage):
+
+            match = re.match(r"(\d+(\.\d+)?)", dosage)  # Matches digits and optional decimal part
+            if match:
+                return float(match.group(1))
+            raise serializers.ValidationError({"dosage": "Invalid dosage format."})
+
```

Comment:
This validation is not required as it's already handled by `DenominationValidator` which is used by dosage-like fields.

Reply:
So should I only remove the validation error part?

Reply:
Keep only validations mentioned in the issue.

Reply:
Fine will do that.

Reply:
Done that, please review it again.Thanks